handlers/grafana_handlers.go

```diff
@@ -26,6 +26,17 @@ func (h *Handler) GrafanaConfigHandler(w http.ResponseWriter, req *http.Request,
 	// 	return
 	// }
 
+	if req.Method == http.MethodGet {
+		err := json.NewEncoder(w).Encode(prefObj.Grafana)
+		if err != nil {
+			logrus.Errorf("error marshaling Grafana config: %v", err)
```

Comment:
We're ready for MeshKit errors. It's time to familiarize with new.error() and begin using MeshKit errors throughout Meshery's codebase.

Reply:
@kumarabd did you make an attempt on the contributor instructions for this? // @DelusionalOptimist